apps/gitness/src/utils/repo-branch-rules-utils.ts

```diff
@@ -30,16 +36,33 @@ const ruleIds = [
   BranchRuleId.DELETE_BRANCH
 ]
 
+const getDetailsByIds = (ids: number[], objMap: { [key: string]: TypesPrincipalInfo }): MultiSelectOption[] => {
+  return ids
+    .map(id => objMap[id])
+    .filter(user => user !== undefined)
```

Comment:
nit: this achieves the same thing:

```suggestion
    .filter(Boolean)
```